### DISCUSSION TITLE
How do I plot an arrow with a length proportional to the magnitude of a force/torque that points along the force/torque

### DISCUSSION THREAD

**Discussion by handsomir** [google-deepmind/mujoco#1296] [General]
Dear Colleagues,

I am having a blast with MuJoCo 3.1.1. Thank you, DeepMind and @yuvaltassa and your team!

I would like to visualize an external force applied to a body, a torque generated inside the body, etc. The best example is the contact force in the MuJoCo visualizer.

I have tried to implement the example found here [https://www.roboti.us/forum/index.php?threads/visualize-programmed-perturbations.4091/](url)  However, I am falling short of finding an object equivalent to `viewer` in the example. 

I think I am missing something fundamental and I cannot find it in the documentation.  Can you help me, please? 

Season's greetings! and thank you!

BU

**Comment by yuvaltassa** [comment]
It is called Perturb Force

<img width="248" alt="Screenshot 2024-01-08 at 10 20 55" src="https://github.com/google-deepmind/mujoco/assets/6387526/b91a3721-d29e-4db7-8f57-6457c7c32b12">
